plantswap/components/NavToggle/navtoggle.module.scss

```diff
@@ -0,0 +1,49 @@
+$transition-speed: 200ms;
+
+.navtoggle {
+  display: block;
+  width: 2em;
+  cursor: pointer;
+  background: none;
+  border: none;
+  margin: 0 auto;
+  height: 52px;
+  width: 52px;
+  aspect-ratio: 1/1;
```

Comment:
With aspect-ratio you only have to define one size. instead of  width and height